src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const arrayPopulationStr = [
+  ...document.getElementsByClassName('population'),
+].map((n) => n.textContent);
+
+const arrayPopulationNum = arrayPopulationStr.map((str) => {
+  return Number(
+    str
+      .split('')
+      .filter((element) => element !== ',')
+      .join(''),
+  );
+});
```

Comment:
The conversion of population strings to numbers is handled correctly by removing commas and using Number(). This ensures that the values can be summed correctly.